resources/lang/en/entities.php

```diff
@@ -328,6 +329,8 @@
     'templates_replace_content' => 'Replace page content',
     'templates_append_content' => 'Append to page content',
     'templates_prepend_content' => 'Prepend to page content',
+    'default_template' => 'Default Page Template',
+    'default_template_explain' => "Assign a default template that will be used for all new pages in this book.",
```

Comment:
This should probably include a note on permissions, advising that the page template will only be used if the page creator has view permission for the page template.